Source/Core/InputCommon/ControllerInterface/CoreDevice.h

```diff
@@ -178,6 +182,7 @@ class Device
   int m_id = 0;
   std::vector<Input*> m_inputs;
   std::vector<Output*> m_outputs;
+  int m_handle;
```

Comment:
This really should be its own type, not just an `int`. It can internally just be an `int` of course but please wrap it in a struct so you can't accidentally use it for something else.